src/scripts/main.js

```diff
@@ -1,3 +1,92 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const titles = thead.querySelectorAll('th');
+const names = titles[0];
+const position = titles[1];
+const age = titles[2];
+const salary = titles[3];
+
+const employees = tbody.querySelectorAll('tr');
```

Comment:
The variable 'employees' is being initialized with the current state of the table rows. If the table rows change dynamically, this variable will not reflect those changes. Consider querying the rows inside each sorting function to ensure the list is always up-to-date.